README.md

```diff
@@ -65,7 +65,7 @@ _TODO: Make it work with latest, confirm if there are any issues._
 6. **Install `grpccache-gen`**:
 
    ```
-   go get -u sourcegraph.com/sqs/grpccache/grpccache-gen
+   go get -u sourcegraph.com/sourcegraph/grpccache/grpccache-gen
```

Comment:
I will remove the sqs repo when this is merged. setting a reminder for myself in 3 days to do so if this PR is merged.